docs/installation/DownloadAndInstall.md

```diff
@@ -1,56 +1,205 @@
+to a directory on your system.  In addition, ScubaGoggles requires the OPA
+executable and the Google credentials file.  It is recommended that you create
+a directory that will contain these required files and serve as a location
+for the reports.  You will run the ScubaGoggles setup utility to indicate the
+location of the output directory.
+
+## ScubaGoggles Setup Utility
+The ScubaGoggles setup utility lets you configure the data directory location,
+as well as the locations of the OPA executable and the Google credentials file.
+It is perfectly fine to locate the OPA executable and credentials files in the
+output directory you create.
+
+When you run the setup utility, it will create a configuration file in your
+top-level user directory called `.scubagoggles` (**Note** the leading dot (.)
+in the file name, which indicates a "hidden" file on linux and macOS operating
+systems).
+
+The configuration file contains the following values used by ScubaGoggles when
+running the conformance assessments:
+
+| Name        | Description                                       |
+|-------------|---------------------------------------------------|
+| credentials | Location and name of the Google credentials file. |
+| opa_dir     | Location of the OPA executable.                   |
+| output_dir  | Location of the ScubaGoggles output directory.    |
+
+Run the setup utility with this command:
+
+```shell
+scubagoggles setup
+```
+
+You will be prompted to enter the output directory, location of the OPA
+executable, and the location and name of the Google credentials file.  By
+default, the OPA executable will be downloaded into the directory you specify.
+You do not have to create the Google credentials file before running setup.  If
+the credentials file doesn't exist, you will see a warning indicating that it is
+missing.  You will need to have the credentials file before running the
+conformance assessment.  See the [instructions for creating a credentials
+file](../authentication/AuthenticationMethods.md).
+
+These are sample outputs from running the setup utility before the required
+files are available.  Warnings are shown, but the `.scubagoggles` configuration
+file is still created.
+
+### Windows Example
+
+```
+> scubagoggles setup
+Setup: output directory
+Scubagoggles output directory [C:\Users\userID\scubagoggles]?
+Create directory C:\Users\userID\scubagoggles [Yes/no]?
+  creating: C:\Users\userID\scubagoggles
+  C:\Users\userID\scubagoggles
+Setup: OPA executable directory
+(WARNING): OPA executable not found in PATH
+Location of OPA executable [C:\Users\userID\scubagoggles]?
+  downloading: opa_windows_amd64.exe
+  OPA executable: C:\Users\userID\scubagoggles\opa_windows_amd64.exe
+Setup: Google API credentials file
+Google credentials (JSON) file [C:\Users\userID\scubagoggles\credentials.json]?
+(WARNING): Google credentials file not found in C:\Users\userID\scubagoggles\credentials.json
+```
```

Comment:
Couple of things here:

- This includes multiple prompts 
- Each prompt asks if user would like to continue the path provided or change it, if the goal is to have all the files centrally located in the user's home directory then why provide an option to change the path where this can be installed
- This could also lead to the user saving different setup files in different locations (ex: opa in one place and credentials in another), this defeats the purpose of a centrally allocated setup files folder
- The above given sequence does not work, since the steps to setup the credentials is after this, so there is no credentials file available for the user to save yet
- Also, the user will then have to place the credentials in this folder, its easier to have everything in the project folder itself
- In the above setup process opa is already installed, and then in the next steps there is a link to install opa, this needs to be updated
- all the setup files can be installed later after they have been created and downloaded, without the prompts to simplify the process

Reply:
noted in https://github.com/cisagov/ScubaGoggles/issues/531